### ISSUE TITLE
Follow up on ace library fix to fix Markdown with Emoji issue

### ISSUE THREAD

**Issue by zuzana-vej** [apache/incubator-superset#11475]
As per conclusion in this bug https://github.com/apache/incubator-superset/issues/11331 this enhancement request is just a reminder to wait for a fix in the Ace Editor and validate it fully fixes the bug with Shifting cursor (forward) when editing Markdown with emoji.

**Comment by nytai** [comment]
Have we opened an issue with https://github.com/ajaxorg/ace?

**Comment by zuzana-vej** [comment]
I haven't. @kkucharc have you / if not could you please open one since you discovered issue is with that?

**Comment by mistercrunch** [comment]
Did we check that we're on the latest Ace, sometimes bumping to latest version addresses little bugs like this one.

**Comment by nytai** [comment]
@mistercrunch yes, it appears it's broken in their demo too (as per https://github.com/apache/incubator-superset/issues/11331)

**Comment by mistercrunch** [comment]
Gotcha. I just looked and our `react-ace` is super behind though. It moved from being based on `brace` to something new `ace-builds`.

**Comment by junlincc** [comment]
@kkucharc asked in Ace editor issue about any news in that. Upgrading react-ace to 9.2.0 didn’t help. 

added Markdown component to roadmap inbox 
https://github.com/apache-superset/superset-roadmap/projects/1

**Comment by kkucharc** [comment]
I asked in this issue about any news in the topic: https://github.com/ajaxorg/ace/issues/4142
